pubs/uis.py

```diff
@@ -14,6 +14,7 @@
 from .content import check_file, read_text_file, write_file, system_path
 
 
+DEBUG = False
```

Comment:
Yep. There is no easy way to make exceptions verbose in the usecase tests, since it involves changing the configuration file. We need a way to make gui error message into true exception when debugging test: this is a simple way to do that.